package.json

```diff
@@ -28,17 +29,18 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/deluan/contentful-migrate.git"
+    "url": "git+https://github.com/devlato/contentful-migrate.git"
   },
   "contributors": [
+    "Denis Tokarev",
     "Deluan Quintao",
     "Jerry Ma"
   ],
   "license": "MIT",
   "bugs": {
-    "url": "https://github.com/deluan/contentful-migrate/issues"
+    "url": "https://github.com/devlato/contentful-migrate/issues"
   },
-  "homepage": "https://github.com/deluan/contentful-migrate#readme",
+  "homepage": "https://github.com/devlato/contentful-migrate#readme",
```

Comment:
Watch out for these changes if you merge this PR, they'll mostly need to be put back to what they were before.